src/Config/RuntimeConfigLoader.cs

```diff
@@ -124,6 +125,7 @@ public static JsonSerializerOptions GetSerializationOptions()
             PropertyNamingPolicy = new HyphenatedNamingPolicy(),
             ReadCommentHandling = JsonCommentHandling.Skip,
             WriteIndented = true,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
```

Comment:
Cool that this is available. Essentially resulting in null properties not being written to the config?

Reply:
Yep! I didn't know that that existed, made this change a whole lot simpler 😁